packages/app/src/components/Root/LogoIcon.tsx

```diff
@@ -1,36 +1,6 @@
 import React from 'react';
-import { makeStyles } from '@material-ui/core';
+import DevHubIcon from './logo/DevhubIcon.png';
```

Comment:
was Allieren not able to get an SVG?

Reply:
No, she said she doesn't have the "formal svg" (I'm assuming the BC logo). She provided this png.
I can pester her some more, the svg would be ideal.